lend/src/components/ActionMMModal/Header.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { Row } from "react-bootstrap";
+import colors from "shared/lib/designSystem/colors";
+import styled, { keyframes } from "styled-components";
+import { components } from "../../designSystem/components";
+import { Title } from "../../designSystem";
+import sizes from "../../designSystem/sizes";
+import { fadeIn } from "shared/lib/designSystem/keyframes";
+import { css } from "styled-components";
+
+export const fadeInDisabled = keyframes`
```

Comment:
If you don't end up using the recommended `animation-direction`, you can move this duplicate keyframe into the `shared` dir